source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -417,6 +450,41 @@ export class Carousel extends Container {
             }
         );
     }
+
+    private createCarouselEvent (type : Enums.CarouselInteractionEvent): CarouselEvent
+    {
+        let currentPageId : string | undefined;
+        if (this.currentPageIndex != undefined) {
+            currentPageId = this.getItemAt(this.currentPageIndex).id;
+        }
+        return new CarouselEvent(type, this.id, currentPageId, this.currentPageIndex);   
+    }
+
+    private raiseCarouselEvent(eventType : Enums.CarouselInteractionEvent) {
+        const onCarouselEventHandler = AdaptiveCard.onCarouselEvent;
+        if (onCarouselEventHandler && eventType == Enums.CarouselInteractionEvent.Pagination) {
```

Comment:
Weird to me that you declared Paginationand in public interface, but you didn't raise it to user.

If it's on purpose, maybe remove Pagination from enum, then 
```
private raiseCarouselEvent(eventType : Enums.CarouselInteractionEvent | "pagination")
```

Reply:
User will get Pagination event.